src/backend/distributed/transaction/worker_transaction.c

```diff
@@ -118,6 +118,11 @@ TargetWorkerSetNodeList(TargetWorkerSet targetWorkerSet, LOCKMODE lockMode)
 	foreach(workerNodeCell, workerNodeList)
 	{
 		WorkerNode *workerNode = (WorkerNode *) lfirst(workerNodeCell);
+		if (workerNode->groupId == 0)
```

Comment:
Using `NodeIsCoordinator ` could make it easier to read the function. Otherwise, a comment could be useful.

Reply:
done.